src/js/components/Autocomplete.js

```diff
@@ -112,7 +115,7 @@ class Autocomplete extends Component {
             .then(suggestions => {
 
               // if any suggestion exists, only then add this as final result item
-              if (isShowFullResults && suggestions.length > 0) {
+              if (isShowFullResults && suggestions.length) {
```

Comment:
Since you removed `array.length > 0` check from here, it will consider it true even when your array is empty?

Reply:
When suggestions array empty, which means array.length will return 0 which will false the if condition.